src/python/py/models/builder.py

```diff
@@ -144,6 +144,7 @@ def __init__(self, config, io_dtype, onnx_dtype, ep, cache_dir, extra_options):
         }
 
         # LayerNorm-specific variables
+        epsilon = config.rms_norm_eps if hasattr(config, "rms_norm_eps") else 9.999999747378752e-06
```

Comment:
why 9.999999747378752e-06 instead of smaller value like 1e-8?

Reply:
That was the value used for some exported models. I'll change it to 1e-6.